.github/workflows/test.yaml

```diff
@@ -55,6 +55,19 @@ jobs:
       - name: Running unit tests
         run: make test-unit
 
+      - name: Running integration tests
+        env:
+          GIT_PROVIDER: github
+          # TODO: Change GIT_ORG_NAME to SovereignCloudStack
+          # when repo SovereignCloudStack/cluster-stack will
+          # have correct and stable releases for OpenStack
+          GIT_ORG_NAME: chess-knight
```

Comment:
We can use `SovereignCloudStack` here probably

Reply:
Yes, after you added the `node-images.yaml`  into [this release](https://github.com/SovereignCloudStack/cluster-stacks/releases/tag/openstack-scs-1-27-v2) yesterday. We can change `GIT_ORG_NAME` to `SovereignCloudStack`. I will also slightly adjust integration test, where we test that.